spack_config/blueos_3_ppc64le_ib/spack.yaml

```diff
@@ -3,7 +3,7 @@ spack:
   config:
     install_tree:
       root: $spack/..
-      padded_length: 128
+      # padded_length: 128
```

Comment:
Padding is used to make binary rellocation more robust: when caching binaries, we need to be sure that the initial installation path is long enough to be replaced by any subsequent installation path. So this is necessary.